tests/unit/src/kakarot/instructions/test_system_operations.py

```diff
@@ -79,9 +82,34 @@ async def test_create(
     async def test_create2(
         self, system_operations, contract_account_class, account_registry
     ):
-        await system_operations.test__exec_create2__should_return_a_new_context_with_bytecode_from_memory_at_empty_address(
+        # we store a memory word in memory
+        # and have our bytecode as the memory read from an offset and size
+        # we take that word at an offset and size and use it as the bytecode to determine the expected create2 evm contract address
+        # Word is 0x 11 22 33 44 55 66 77 88 00 00 ... 00
+        # bytecode should be 0x 44 55 66 77
+        memory_word = 22774453838368691922685013100469420032
```

Comment:
fyi this could actually be write `memory_word = 0x11223344556677880000...` without quote (I think that this is copy/pasted from elsewhere)